docs/reference/contributing/guidelines/workflow.md

```diff
@@ -79,6 +79,43 @@ Each feature has a tech lead. This person is responsible for:
 - Rebasing often to track master development.
 - Reviewing any addition to the feature branch (approval required by the feature tech lead or another assigned person).
 
+##### Pull request types
+
+We consider the following pull request types.
+
+##### Fix
+
+A bug fix should be backward-compatible internal change that fixes incorrect behavior. The fix should provide a description what and how it fixes the issue. The test should be included to catch this issue in the future and confirm the correct behavior.
+
+Release: patch version
+
+##### Refactor
+
+Refactors are intended for minor versions if they are not fixing specific issue as they 
```

Comment:
Is it assumed that a refactor is common knowledge here? This described what they're targeted for, but not -what- they are.